### ISSUE TITLE
v0.13.1 autocomplete error for empty query

### ISSUE THREAD

**Issue by creotutar** [yext/answers-search-ui#659]
**Environment**
`dist-v0.13.1`
importing from `answers/dist/answers-umd.js`
Chrome

**Error**
Getting a CORS error in the chrome console for the initial autocomplete request when there is no query in the searchbar.

```
Access to fetch at 'https://liveapi-cached.yext.com/v2/accounts/me/answers/autocomplete?{redacted}' from origin 'http://localhost:9027' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://localhost:9028' that is not equal to the supplied origin. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Steps to recreate**
Navigate to an answers site with a SearchBar, click on the search bar while there is no query. Autocomplete results will not show up.

**Notes**
This does not happen on dist-v0.9.5

**Response**
```
{
meta: {
uuid: "a342f217-0c2c-4f3c-b5b7-4a0aab875241",
errors: [ ]
},
response: {
input: {
value: "",
queryIntents: [ ]
},
results: [
{
value: "Locations near me",
matchedSubstrings: [ ],
queryIntents: [
"NEAR_ME"
]
},
{
value: "Locations in Sacramento",
matchedSubstrings: [ ],
queryIntents: [ ]
},
{
value: "Open a savings account",
matchedSubstrings: [ ],
queryIntents: [ ]
},
{
value: "What is my routing number?",
matchedSubstrings: [ ],
queryIntents: [ ]
},
{
value: "Set up account alerts",
matchedSubstrings: [ ],
queryIntents: [ ]
},
{
value: "How to use direct deposit?",
matchedSubstrings: [ ],
queryIntents: [ ]
}
]
}
}
```

**Comment by rosiegrant** [comment]
oo ok christian i had the same issue testing locally -- my browser was caching the version before the upgrade. a hard refresh fixed it. this looks ok on staging, could you confirm?

**Comment by creotutar** [comment]
ah yes! that is very weird as it was giving the liveapi-cached url as erroring (which was not available pre v0.13.1 iirc)

i can confirm it is now working on ie11

**Comment by rosiegrant** [comment]
thanks!
